allennlp/models/coreference_resolution/coref.py

```diff
@@ -586,40 +602,49 @@ def decode(self, output_dict: Dict[str, torch.Tensor]):
             which are in turn comprised of a list of (start, end) inclusive spans into the
             original document.
         """
-        top_spans = output_dict["top_spans"].data.cpu()
-        predicted_antecedents = output_dict["predicted_antecedents"].data.cpu()
+        batch_top_spans = output_dict["top_spans"].data.cpu()
```

Comment:
Shape comments here would be helpful.

Reply:
Also saying _what_ is in here, not just the shape, would be nice.  I _think_ it's this: `batch_top_spans` has shape `(batch_size, num_spans, 2)`, where it holds, for each span, the start and end index of the span.  `batch_predicted_antecedents` has shape `(batch_size, num_spans)`, and each element is an index into `num_spans`, saying what the span's antecedent is.  `antecedent_indices` is...?